specification/source/cdl.rst

```diff
@@ -1688,118 +1688,75 @@ Therefore, tools that process CDL can infer the following information:
 
 .. _sec_tag_pro:
 
-Tagged Properties
+Semantic Information
 .................
 
-The buildings industry uses different tagging schemes such as
-Brick (https://brickschema.org/) and Haystack (https://project-haystack.org/).
-CDL allows, but does not require, use of the Brick or Haystack tagging scheme.
+The buildings industry uses different metadata schemes such as
+`Brick <https://brickschema.org/>`_, `Project Haystack <https://project-haystack.org/>`_ and `ASHRAE 223p <https://www.ashrae.org/about/news/2018/ashrae-s-bacnet-committee-project-haystack-and-brick-schema-collaborating-to-provide-unified-data-semantic-modeling-solution>`_. CDL allows, but does not require, use of the Brick or Haystack or any other tagging scheme.
 
-CDL allows to add tags to declarations that instantiate
+* Semantic information will be included within the CDL ``annotation`` keyword, using the ``__semantic`` annotation. Every element in CDL can have an annotation and correspondingly, semantic information
```

Comment:
I know we agree to use `__semantic` but this can cause a name conflict if some other library also adds `__semantic` but using their own definition. The specification of vendor annotation is here:
https://specification.modelica.org/v3.4/Ch18.html
Based on that, on second thoughts I think we should call it something like `__Buildings(semantic(...))`, where `Buildings` is the name of the library, which I chose to avoid putting `__LBNL(...)` into the standard. This should keep us clear of name clashes, and `modelica-json` can search for `__"name of library"(semantic(...))`. I also selected `__Buildings` as I think use of `__ASHRAE_...` would need permission, and is not suited for other semantic annotation such as for equipment that may be in Brick or Haystack, or plain text.

So the formal specification would be
```
__"name of the library"(semantic(...))
```
where "name of the library" is the name of the top-level package, such as `Buildings` for sequences that are stored in the `Buildings` library.  If a user develops an own library of sequences that may be called "MyControlLibrary" then the annotation can either be ``__MyControlLibrary(semantic(...))`` or ``__Buildings(semantic(...))`` because a tool, such as modelica-json, that parses the semantic information can look for both of these strings. Hence, sequences copied from a library such as `Buildings` into a user library such as `MyControlLibrary` can retain the semantic information as-is, or they can optionally rename it and add their own semantic information.

Reply:
@anandkp92 : Actually, `__MyControlLibrary(semantic(...))` can give the same name clash as we want to avoid with abandoning `__semantic`. So I think the options should be either of these two
- `__Buildings(semantic(...))` in the Buildings library, and also allow
- `__CDL(semantic(...))`.

We can then use `__CDL` in the CDL sequences, and `__Buildings` everywhere else, and modelica-json will treat them the same. I think that will work unless you have concerns, because we are in control of both names, and the ASHRAE standard would not need to be tied to the Buildings library.

Reply:
I have updated the specification to use __Buildings and __cdl (to be consistent with rest of CDL annotations)